tftb/tests/test_amplitude_modulations.py

```diff
@@ -2,6 +2,9 @@
 import numpy as np
 from numpy import pi
 from scipy.signal import argrelmax
+# diagnostics:
+import base		# this would be the 'local' import, so it will depend on the local directory structure. nominally, if we properly configure setup.py
+					# and install, we should be able to use the line below. are tests meant to be part of the package, or run separately?
```

Comment:
They're meant to be separate. Simply using nose takes care of this. The following import works with `nosetests`.